deformable_manipulation_experiment_params/launch/cloth_single_pole_params.launch

```diff
@@ -29,21 +32,22 @@
     <param name="table_surface_z"       value="$(arg table_surface_z)"  type="double"/>
 
     <!-- Cloth size params -->
-    <arg name="cloth_x_size"                    value="0.3"/>
-    <arg name="cloth_y_size"                    value="0.5"/>
-    <arg name="cloth_num_control_points_x"      value="30"/>
-    <arg name="cloth_num_control_points_y"      value="50"/>
+    <arg name="cloth_x_size"                    default="0.3"/>
+    <arg name="cloth_y_size"                    default="0.4"/>
+    <arg name="cloth_num_control_points_x"      default="15"/>
+    <arg name="cloth_num_control_points_y"      default="20"/>
 
     <param name="cloth_x_size"                  value="$(arg cloth_x_size)"                 type="double"/>
     <param name="cloth_y_size"                  value="$(arg cloth_y_size)"                 type="double"/>
     <param name="cloth_num_control_points_x"    value="$(arg cloth_num_control_points_x)"   type="int"/>
     <param name="cloth_num_control_points_y"    value="$(arg cloth_num_control_points_y)"   type="int"/>
 
     <!-- Cloth position params -->
-    <arg name="cloth_leading_edge_x"    value="0.06"/>
```

Comment:
Same here, we want the parameters to stay the same as seen in the paper for now. Once that is out the door, we can change these, but they probably shouldn't be changed in the master branch.